src/types.ts

```diff
@@ -4119,6 +4119,13 @@ export type EventHook = {
   sns_key?: string;
   sns_secret?: string;
   sns_role_arn?: string;
+
+  // pending message config
+  timeout_ms?: number;
+  callback?: {
+    mode?: 'CALLBACK_MODE_NONE' | 'CALLBACK_MODE_REST' | 'CALLBACK_MODE_TWIRP';
```

Comment:
```suggestion
    mode: 'CALLBACK_MODE_NONE' | 'CALLBACK_MODE_REST' | 'CALLBACK_MODE_TWIRP';
```
I'd make this inner `mode` property required otherwise you'd be able to pass empty object which seems pointless. If this is response I'd imagine you guys mark this field with `omitempty`.